### PR TITLE
Flask Cleanup

### PR DESCRIPTION
**Pull request by streetcoder123**
Tasks:
- https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30041
- https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30111
- https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30311
- https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30312

**Changes**
- Convert all comments to full sentences with a period at the end.
- Remove parenthesis around all conditionals (more python way)
-  Convert all HTTP status codes to constants to be more intuitive
- Abstract away the logic of making responses out of mongo and sql to server files.

**How to Check**
- Generate Flask applications - without cosmos, with cosmos (sql and mongo) and check if they work appropriately.
- Check for any code quality issues.

GitHub Issue #810

### CODE REVIEW

**Review by jasmingoh — COMMENTED** [review]
I can't comment on the line, but https://github.com/microsoft/WebTemplateStudio/pull/831/files#diff-9f13f2c1864f8a3d764e75d8308c68bbR10 seems to be missing a period at the end

**Comment by streetcoder123** [comment]
@jasmingoh Thanks. I will update it.

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/mongo_service.py [hunk 1]
```diff
@@ -8,19 +8,17 @@
 def get():
     items = list_items.find()
     serialized_list_items = [serialize(item) for item in items]
-    return jsonify(serialized_list_items)
+    return serialized_list_items
```
Comment (jsondoo) [inline templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/mongo_service.py:11]: Just wondering why are we removing jsonify?

Comment (jsondoo) [reply templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/mongo_service.py:11]: oh wait never mind, looked below

**Review by jasmingoh — APPROVED** [review]

**Review by Tanya0609 — APPROVED** [review]

**Review by piotrmark — APPROVED** [review]

**Pull request merged**